client/src/pages/reports/Show.js

```diff
@@ -376,6 +374,14 @@ const ReportShow = ({ setSearchQuery, pageDispatchers }) => {
             {canEmail && (
               <Button onClick={toggleEmailModal}>Email report</Button>
             )}
+            <Button
+              value="compactView"
+              type="button"
+              bsStyle="primary"
+              onClick={onCompactClick}
+            >
+              {Settings.fields.report.compactView}
```

Comment:
It is assumed here that `fields.report.compactView` is defined, however the yml schema does not require it. Either make the field required or provide a default value for it.

Reply:
I wasn't awar of the required fields in that schema, changed!